cmd/ksync/ksync.go

```diff
@@ -130,6 +130,21 @@ func remoteFlags(flags *pflag.FlagSet) {
 
 		log.Fatal(err)
 	}
+
+	flags.String(
+		"graph-root",
```

Comment:
I'd prefer that this is named `docker-root` instead of graph (since that's dependent on a single driver).